library/commandline/src/modules/CommandLine.rb

```diff
@@ -1477,6 +1441,7 @@ def Run(commandline)
       return !Aborted() if !Init(commandline, WFM.Args)
 
       ret = true
+      read_only = true # do finish action only if module is not read only
```

Comment:
This seems to mean that the default is read-only, which will IMHO result in bug reports "why is my config not changed?!".

Reply:
no, default for any operation that does not define key is write. So this is default if there is no operation, it is read_only and if there is operation that does not have readonly flag set to true, it is set to false.

Reply:
No operation? But there is always an operation, otherwise I get the GUI!?